info/v1/container.go

```diff
@@ -516,6 +516,31 @@ type FsStats struct {
 	WeightedIoTime uint64 `json:"weighted_io_time"`
 }
 
+// NvidiaGpuStats are measured using NVML.
+type NvidiaGpuStats struct {
+	// Minor number of the GPU device (/dev/nvidia[minor_number])
+	MinorNumber uint64 `json:"minor_number"`
+
+	// Product name of the GPU
+	Name string `json:"name"`
+
+	// Total installed GPU memory.
+	// unit: bytes
+	MemoryTotal uint64 `json:"memory_total"`
+
+	// Total GPU memory allocated.
+	// unit: bytes
+	MemoryUsed uint64 `json:"memory_used"`
+
+	// Percent of time over the past sample period (of nvml) during which
+	// one or more kernels was executing on the GPU.
+	Busy uint64 `json:"busy"`
+
+	// Power usage for this GPU and its associated circuitry.
+	// unit: milliwatts
+	PowerUsage uint64 `json:"power_usage"`
```

Comment:
Is this cumulative?  if so, we should specify.  If not, what is the period of time?

Reply:
The unit is milliwatts. The  [documentation](http://docs.nvidia.com/deploy/nvml-api/group__nvmlDeviceQueries.html#group__nvmlDeviceQueries_1g7ef7dff0ff14238d08a19ad7fb23fc87) doesn't say how long they sample the energy consumption to calculate the power usage. I would assume it's another point-in-time gauge.